test/IntegrationTests/MSTest.VstestConsoleWrapper.IntegrationTests/SuiteLifeCycleTests.cs

```diff
@@ -167,6 +167,7 @@ private void ValidateTestRunLifecycle(string targetFramework)
             {(targetFramework == "net6.0"
                 ? "Console: LifeCycleClassCleanup.DisposeAsync was called\r\nConsole: LifeCycleClassCleanup.Dispose was called"
                 : "Console: LifeCycleClassCleanup.Dispose was called")}
+            Console: LifeCycleClassCleanup.ClassCleanup was called
```

Comment:
This seems to indicate that some cleanup was changed, is it expected? If so, please try to help review by adding some comments helping to understand the changes so that reviewer isn't forced to check it up.

Reply:
yes changed because becouse this class has the default behavior which is `EndOfClass` now and before was `EndOfAssembly`